include/nbl/builtin/glsl/blit/normalization/normalization.glsl

```diff
@@ -20,10 +20,10 @@ uint integerDivide_64_32_32(in uint dividendMsb, in uint dividendLsb, in uint di
 #define scratchShared _NBL_GLSL_SCRATCH_SHARED_DEFINED_
 
 #include <nbl/builtin/glsl/blit/parameters.glsl>
-nbl_glsl_blit_parameters_t nbl_glsl_blit_normalization_getParameters();
+nbl_glsl_blit_parameters_t nbl_glsl_blit_getParameters();
 
-nbl_glsl_blit_normalization_pixel_t nbl_glsl_blit_normalization_getPaddedData(in ivec3 coord);
-void nbl_glsl_blit_normalization_setData(in nbl_glsl_blit_normalization_pixel_t data, in ivec3 coord);
+vec4 nbl_glsl_blit_normalization_getPaddedData(in ivec3 coord);
```

Comment:
dont have a "paddedData" method, I said to hoist the "out of bounds" check outside

Reply:
Yes its done, yet to commit i.e. comes in the next commit.

Reply:
Resolved.